pep-0008.txt

```diff
@@ -1259,10 +1293,9 @@ Programming Recommendations
   any return statement returns an expression, any return statements
   where no value is returned should explicitly state this as ``return
   None``, and an explicit return statement should be present at the
-  end of the function (if reachable).
-
-  Yes::
+  end of the function (if reachable)::
 
+      # Correct:
```

Comment:
Here I also think we could use an extra blank line after the Correct/Wrong comments.

Reply:
Ok